api-clients/JSClient.html

```diff
@@ -0,0 +1,43 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+    <script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-2.0.3.min.js"></script>
+</head>
+<body>
+    <h2>List of Products</h2>
+    <div>
+        <ul id="products" />
+    </div>
+
+    <h2> List of Categories</h2>
+    <div>
+        <ul id="categories">
+    </div>
+</body>
+</html>
+<script>
+
+    let productsApiUrl = 'https://localhost:7087/api/products';
```

Comment:
Let's convert it to `const` and extract base domain + part as separate constant to be able to easily modify it.